Documentation/index.rst

```diff
@@ -4,6 +4,14 @@
     Please use the official rendered version released here:
     https://docs.cilium.io
 
+.. versionadded::HEAD
+.. note::
+
+    WARNING: This documentation is for the development version and may not be stable.
+    For the latest stable release, please refer to the `official documentation`_
+    
+.. _official documentation: https://docs.cilium.io/en/stable/
```

Comment:
This note will show up

1. Only on the index page
2. For all branches, once this lands into support branches.

This is not what we want.

Reply:
I was also unsure about the implementation, thanks! i think i should delete this pr